site/content/docs/main/troubleshooting/common-errors.md

```diff
@@ -0,0 +1,65 @@
+# Troubleshooting common errors
```

Comment:
I see we have `access logs` based debugging which is really useful but should we also mention prometheus stats?

Reply:
yep, will add

I hope this can grow over time but if you have any other ideas for types of errors to record and debugging strategies for those we can tackle it in this PR or a follow up